server/base/src/main/java/org/apache/accumulo/server/client/BulkImporter.java

```diff
@@ -175,78 +175,85 @@ public void run() {
 
       Map<Path,Integer> failureCount = new TreeMap<>();
 
-      for (Entry<Path,List<KeyExtent>> entry : assignmentFailures.entrySet())
-        failureCount.put(entry.getKey(), 1);
-
-      long sleepTime = 2 * 1000;
-      while (!assignmentFailures.isEmpty()) {
-        sleepTime = Math.min(sleepTime * 2, 60 * 1000);
-        locator.invalidateCache();
-        // assumption about assignment failures is that it caused by a split
-        // happening or a missing location
-        //
-        // for splits we need to find children key extents that cover the
-        // same key range and are contiguous (no holes, no overlap)
-
-        timer.start(Timers.SLEEP);
-        sleepUninterruptibly(sleepTime, TimeUnit.MILLISECONDS);
-        timer.stop(Timers.SLEEP);
-
-        log.debug("Trying to assign {} map files that previously failed on some key extents",
-            assignmentFailures.size());
-        assignments.clear();
-
-        // for failed key extents, try to find children key extents to
-        // assign to
+      int retries = context.getConfiguration().getCount(Property.TSERV_BULK_RETRY);
+      if (retries == 0) {
+        log.warn("Retries set to 0. All failed map file assignments will not be retried.");
```

Comment:
Minor grammatical improvement. (Kein vs. Nicht)

```suggestion
        log.warn("Retries set to 0. No failed map file assignments will be retried.");
```

Reply:
Also, this should probably be logged at `ERROR` instead of `WARN`, since this log message is replacing the individual messages for each file, which currently logs at `ERROR`.

Reply:
Changed in latest commit.